src/background/index.js

```diff
@@ -0,0 +1,11 @@
+/* globals chrome */
```

Comment:
We can move this to the ESLint config once https://github.com/prettier/prettier-browser-extension/pull/352 is merged.